packages/mobile-components/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+
+export type ButtonProps = {
+ onPress: () => void;
+ text: string;
+ color?: string;
+ textColor?: string;
+};
+
+const styles = StyleSheet.create({
+    button: {
+        paddingVertical: 8,
+        paddingHorizontal: 16,
+        borderRadius: 4,
+        alignSelf: "flex-start",
+        flexGrow: 0,
+        backgroundColor: "#005ea2",
+    },
+    buttonText: {
+        color: "white",
+        fontSize: 16,
+        fontWeight: "bold",
+    },
+    buttonContainer: {
+        alignItems: "flex-start",
+        flex: 1,
+    },
+});
+
+export const Button = ({ text, onPress, color, textColor }: ButtonProps) => (
+ <View style={styles.buttonContainer}>
```

Comment:
This is just a simple react-native button to demonstrate that it renders in Storybook:

![Screenshot 2025-02-17 at 4 35 36 PM](https://github.com/user-attachments/assets/34a37c36-cc74-4800-b691-b32d7ef39567)